### DISCUSSION TITLE
Is there a shortcut to open the control surfaces window?

### DISCUSSION THREAD

**Discussion by lxne** [CommandPost/CommandPost#3389] [Technical Support] [answered]
Hi, I would like to have a keyboard shortcut or a button on my devices to open the control surfaces window.
Best thing would be if the shortcut could switch to the correct tab, too.
E.g. pressing the button on a loupedeck takes you to control surfaces - loupedeck
Is there an easy way to do this?

**Comment by latenitefilms** [comment, marked as answer]
There's a built-in action for "Open Control Surfaces", and also an action called "Loupedeck" to open directly to the Loupedeck page. Basically every panel SHOULD have a built-in action for this.

**Reply by lxne** [reply]
Thank you so much. This was the kickstart I needed!

Best wishes
